src/pages/PoolFinder/index.tsx

```diff
@@ -136,7 +143,7 @@ export default function PoolFinder() {
             </Row>
           ) : (
             <Text fontWeight={500} fontSize={20} marginLeft={'12px'}>
-              Select a Token
+              <Trans id="currencies.search.selectToken">Select a token</Trans>
```

Comment:
Capitalization changed?

Reply:
That oe was intentional. The string 'Select a token' is used throughout the app so I just standardized it.